devops/helm/crunchy-postgres/templates/PostgresCluster.yaml

```diff
@@ -70,6 +70,10 @@ spec:
       databases:
         - {{ template "crunchy-postgres.fullname" . }}
       options: "CREATEROLE"
+    - name: {{ .Values.readonlyuser }}
+      databases:
+        - {{ template "crunchy-postgres.fullname" . }}
```

Comment:
I would strongly recommend that the code and scripts here are exactly executed and tested along with the access of read-only user in our current operating DEV environment. 

I see that has not been done in DEV(not the new DEV). 

I know that we have a sync-up planned and expecting the setup to happen in DEV post sync up.

Reply:
Thanks. I was able to test with rad-only-user in current dev.